src/test/java/GoogleSearchTest.java

```diff
@@ -33,21 +36,21 @@ public void setup() {
     options.addArguments("--disable-dev-shm-usage");
     options.addArguments("--headless");
     driver = new ChromeDriver(options);
+    google  = new GooglePage(driver);
   }
 
   @Test (description = "This test is in charge of redirecting "
       + "to the google website and searching for an specified subject")
   public void searchAtGoogle() {
     Reporter.log("The chrome version being used is the latest available on the linux repo");
-    driver.get("https://www.google.com/");
-    driver.findElement(By.name("q")).sendKeys("mjolnir" + Keys.RETURN);
+    google.goToGoogle();
+
+    google.search("Mjolnir");
     Reporter.log("First search task completed");
-    driver.findElement(By.name("q")).clear();
-    driver.findElement(By.name("q")).sendKeys("perficient"
-        + Keys.RETURN);
+
+    google.search("perficient");
     Reporter.log("Second search task completed");
-    driver.findElement(By.id("rso")).findElements(By.xpath("/*"))
-      .get(0).findElement(By.tagName("h3")).click();
+
     Assert.assertTrue(driver.getTitle().contains("perficient"));
```

Comment:
Into a test you should not call selenium methods, I mean, all calls should be encapsulated into page object models